### DISCUSSION TITLE
how to create this image

### DISCUSSION THREAD

**Discussion by nhemrado** [lllyasviel/Fooocus#2823] [General]
![Screenshot 2024-04-27 082556](https://github.com/lllyasviel/Fooocus/assets/164300419/6dfc4476-6d64-4587-aa3c-945c75892d9e)

Dear all, 
I have try serval to create that kind of image but can not make it. can you tell  how to do that kind of images ?
thank you.

**Comment by xtiondiv123** [comment]
![1714272694353](https://github.com/lllyasviel/Fooocus/assets/152345887/52f7ac53-93da-488b-85c6-20b1b1da7b3a)
i think you can add some words for describe the picture then you want， i write a boy sit on the ground ， war background ，he hold a cake， i am not sure the prompt can create the people‘s part of body，you can use PS software solve it
